Package.swift

```diff
@@ -29,7 +29,7 @@ let package = Package(
     // Main SwiftNIO package
     .package(url: "https://github.com/apple/swift-nio.git", from: "2.14.0"),
     // HTTP2 via SwiftNIO
-    .package(url: "https://github.com/apple/swift-nio-http2.git", from: "1.8.0"),
+    .package(url: "https://github.com/apple/swift-nio-http2.git", from: "1.12.0"),
```

Comment:
Mind bumping this to `1.12.1`? We missed something the first time with the window size: https://github.com/apple/swift-nio-http2/pull/205